### PR TITLE
release: sourcegraph@5.2.1

### PR DESCRIPTION
**Pull request by keegancsmith**
This pull request is part of the Sourcegraph 5.2.1 release.

* [Release batch change](https://sourcegraph.sourcegraph.com/organizations/sourcegraph/batch-changes/release-sourcegraph-5.2.1)
* [Tracking issue](https://github.com/sourcegraph/sourcegraph/issues/57319)

### Test plan

CI checks in this repository should pass, and a manual review should confirm if the generated changes are correct.

### CODE REVIEW

File: configure/embeddings/embeddings.Deployment.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ spec:
     spec:
       containers:
         - name: embeddings
-          image: index.docker.io/sourcegraph/embeddings:insiders@sha256:b68cade0fdf2afcd8e78fbcb44d25bcecaec448f0c6e29fa716834cb95aa8a47
+          image: index.docker.io/sourcegraph/embeddings:insiders@sha256:6bfffb3f0d15a4f71d70989656a35d617021574b9081304602e310436ce9f3f9
```
Comment (keegancsmith) [inline configure/embeddings/embeddings.Deployment.yaml:31]: for follow-up: is it write that we don't deploy a tagged version here? cc @stefanhengl

File: configure/executors/k8s/executor.Deployment.yaml [hunk 2]
```diff
@@ -30,7 +30,7 @@ spec:
       serviceAccountName: executor
       containers:
         - name: executor
-          image: index.docker.io/sourcegraph/executor-kubernetes:insiders@sha256:0644f1e165fe3dac6cfea1f36a14596fd52d88200385d506d2746b6d522029c3
+          image: index.docker.io/sourcegraph/executor-kubernetes:insiders@sha256:83255544f6837576419008008f7a4b7c19cad1831001ba7176311930539d3d9e
```
Comment (keegancsmith) [inline configure/executors/k8s/executor.Deployment.yaml:33]: for follow-up: @eseliger is it alright that we don't deploy a tagged image here?

File: configure/embeddings/embeddings.Deployment.yaml (hunk 1 above)
Comment (camdencheek) [reply configure/embeddings/embeddings.Deployment.yaml:31]: This should definitely have a tag. Maybe we need to update the list of images?

**Review by camdencheek — APPROVED** [review]
stamping pending resolution of untagged images

**Comment by keegancsmith** [comment]
I will follow up with tags. Indeed the actual versions used where incorrect as well.

**Comment by keegancsmith** [comment]
https://github.com/sourcegraph/deploy-sourcegraph/pull/4277

**Pull request merged**